languages/CookieConsent/cs.ini

```diff
@@ -0,0 +1,32 @@
+Accept All Cookies = "Přijmout vše"
+Accept Only Essential Cookies = "Přijmout pouze nezbytné cookies"
+analytics_cookies_description_html = "<p>Analytické cookies umožňují sledovat souhrnné informace o&nbsp;návštěvnosti stránek.<br>Díky těmto informacím jsme schopni lépe přizpůsobit stránky uživatelům webu.</p><p>Jsou povoleny pouze na&nbsp;základě vašeho souhlasu.</p>"
```

Comment:
"<p>Analytické cookies umožňují sledovat souhrnné informace o&nbsp;chování návštěvníků stránel.<br>Díky těmto informacím jsme schopni lépe přizpůsobit stránky uživatelům webu.</p><p>Jsou povoleny pouze na&nbsp;základě vašeho souhlasu.</p>"

Reply:
Fixed. But just keep "Vašeho" instead "vašeho" you proposed - for consistency.